sources/engine/Stride.Engine/Engine/FlexibleProcessing/IProcessorBase.cs

```diff
@@ -2,11 +2,14 @@
 // Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
 
 using Stride.Core;
+using Stride.Engine.Design;
 
 namespace Stride.Engine.FlexibleProcessing
 {
     public interface IProcessorBase
     {
+        /// <summary> The logic using this value expects it to be constant at runtime </summary>
+        ExecutionMode ExecutionMode => ExecutionMode.Runtime;
```

Comment:
Do we want the default to act like a regular `EntityProcessor` for the sake of consistency?

Reply:
I think most users would actually expect their processors to only run at runtime out of the box, like script do. I understand that it's a bit counter intuitive when you come from how `EntityProcessor` work out of the box, which is why I wanted to change that as well but that's a far bigger breaking change

Reply:
With the amount of times I have crashed the editor due to me forgetting to add `ExecutionMode` I fully agree with that logic. 

Sounds good to me then.